src/main/java/com/whalespottingjava/models/validation/validators/LongitudeValidator.java

```diff
@@ -0,0 +1,12 @@
+package com.whalespottingjava.models.validation.validators;
+
+import com.whalespottingjava.models.validation.annotations.LongitudeConstraint;
+import jakarta.validation.ConstraintValidator;
+import jakarta.validation.ConstraintValidatorContext;
+
+public class LongitudeValidator implements ConstraintValidator<LongitudeConstraint, Double> {
+    @Override
+    public boolean isValid(Double value, ConstraintValidatorContext context) {
+        return value != null && value <= 180 && value >= -180;
```

Comment:
it is more a question. I know why you are checking 180 and -180. But should it be more explicit? I mean for longitude it is logical that it is btw those value , in case of more complex case it would be good to have a explicit name for these number/variable/constant